### PR TITLE
Connect Cron with Twilio

### PR DESCRIPTION
**Pull request by KirstynNatavio**

### CODE REVIEW

**Comment by KirstynNatavio** [comment]
@AryanJ-NYC

**Comment by KirstynNatavio** [comment]
@DwayneJ23

File: routes/index.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
 var express = require('express');
 var router = express.Router();
 
+var cron = require('node-cron');
+
```
Comment (AryanJ-NYC) [inline routes/index.js]: Replace all `var`s here with `const`s.

File: routes/index.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+	res.render('index', { title: 'Tyme' });
+	delayMessage('+13215279114',"IT WOOOOORRRRRKSS");
 });
 
+function delayMessage(number, message) {
+	const accountSid = 'AC65bdbdfbf0837bccd4962a2293745ceb';
```
Comment (AryanJ-NYC) [inline routes/index.js:13]: Indentation.

File: routes/index.js [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+    const authToken = '47d2cd5dc2994c6824db3ea677586b5d';
+    const client = require('twilio')(accountSid, authToken);
+    cron.schedule('5 * * * * *', function() {
+		client.messages
+		.create({
+			to: number,
+			from: '+12012926280',
+			body: message,
+     	})
```
Comment (AryanJ-NYC) [inline routes/index.js]: Indentation. ☝️

File: routes/index.js [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
 var router = express.Router();
 
+var cron = require('node-cron');
+
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+	res.render('index', { title: 'Tyme' });
+	delayMessage('+13215279114',"IT WOOOOORRRRRKSS");
 });
```
Comment (AryanJ-NYC) [inline routes/index.js:10]: Two space indentation.

**Review by AryanJ-NYC — CHANGES_REQUESTED** [review]

**Review by AryanJ-NYC — APPROVED** [review]

**Pull request merged**